src/helpers/prerenderHelper/getSitemapEntries.ts

```diff
@@ -71,15 +71,21 @@ export const getSitemapEntries = (): SitemapEntry[] => {
   };
 
   const searchEntry: SitemapEntry = {
```

Comment:
I think you can completely remove search from sitemap and even ignore from robots.txt: can be done in a separated PR